NEXT_CHANGELOG.md

```diff
@@ -27,6 +27,21 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 
 ## ❗ BREAKING ❗
 
+### Homepage ([PR #1768](https://github.com/apollographql/router/pull/1768))
+
+The router now displays a homepage by default, instead of the studio sandbox.
+
+You can still disable the homepage, and enable the sandbox by editing your configuration:
```

Comment:
I think we should frame this a bit differently because it doesn't justify the (somewhat dramatic, but intentional!) change to the user-story.  We can tie it to the story that is further down in the CHANGELOG that adds the `--dev` flag.  We don't need to actually do that in this PR though — we can do it wholistically once we have everything on `main` and using the Release PR.  What do you think?

Reply:
Totally agreed, shall we edit this together here, or would you rather have a nice pass at that while making the release pr ?

Reply:
I got it! 😄